topics/single-cell/tutorials/scrna-ncbi-anndata/tutorial.md

```diff
@@ -0,0 +1,516 @@
+
+Unfortunately, the names of the files don't exactly match the data in our excel sheet. This can make finding the right samples a bit difficult! A summary of which samples match which file can be found below:
+
+- **AUG_PB1A** ---> **PA_AUG_PB_1A**
+- **AUG_PB1B** ---> **PA_AUG_PB_1B**
+- **MAY_PB1A** ---> **PA_PB1A**
+- **MAY_PB1B** ---> **PA_PB1B**
+- **MAY_PB2A** ---> **PA_PB2A**
+- **MAY_PB2B** ---> **PA_PB2B**
+
+You may also notice that we have multiple sample files with the same name suffixed with `_Pool_X`. These are replications where multiple samples are taken of the same tumor area in order to get a more comprehensive view of the gene data of the tumor. We will need to combine these replication files during processing.
+
+<div class="Manual" markdown="1">
+
+We now need to find the files relating to each patient. Selecting the output {% icon param-file %} **Unzip** tool shows you 53 files in a {% icon param-collection %}. We will work with the 10 files associated with our patient.
+
+> <hands-on-title>Unhiding 10 target files</hands-on-title>
+>
+> 1. In the {% icon galaxy-history %} Galaxy history panel, select the *Include hidden* icon {% icon galaxy-show-hidden %} to see the 53 files in your dataset collection.
+>
+> 2. Scroll down to visualise the following 10 samples (likely in opposite order to this!):
+>    ```
+>    GSM5353214_PA_AUG_PB_1A_S1.dge.txt
+>    GSM5353215_PA_AUG_PB_1B_S2.dge.txt
+>    GSM5353216_PA_PB1A_Pool_1_3_S50_L002_dge.txt
+>    GSM5353217_PA_PB1A_Pool_2_S107_L004_dge.txt
+>    GSM5353218_PA_PB1B_Pool_1_2_S74_L003_dge.txt
+>    GSM5353219_PA_PB1B_Pool_2_S24_L001_dge.txt
+>    GSM5353220_PA_PB1B_Pool_3_S51_L002_dge.txt
+>    GSM5353221_PA_PB2A_Pool_1_3_S25_L001_dge.txt
+>    GSM5353222_PA_PB2B_Pool_1_3_S52_L002_dge.txt
+>    GSM5353223_PA_PB2B_Pool_2_S26_L001_dge.txt
+>    ```
+> 3. For each of those 10 samples, select the *Unhide* icon {% icon galaxy-show-hidden %}. This will move them from {% icon galaxy-show-hidden %} *Hidden* to *Active*.
+>
+> 4. Navigate back to the {% icon galaxy-show-active %} *Active* datasets list, where you will now see your 10 target samples.
+{: .hands_on}
+</div>
+
+# Converting to AnnData and Combining Samples
+
+The first step is to convert all of the raw files into AnnData objects, this can be done one at a time or all at once by selecting ```multiple datasets``` when chosing the input and highlighting all the raw data files.
+
+> <tip-title>Adding Metadata</tip-title>
+> Before starting to process the data it is generally a good idea to add tags to each sample in order to keep track of what data is being processed, below is some of the imported data with added tags for the patient and sample id:
+>
+> ![Imported data with tags](../../images/scrna-ncbi-anndata/metadata.png "Imported data with tags")
```

Comment:
We figured this out by looking at (the paper, the spreadsheet tabs x and y, etc.....). It was not particularly easy, and it's a good reminder that careful and clear metadata when you upload data is crucial!